### PR TITLE
Updated the careers page design

### PR DESCRIPTION
**Pull request by Nikhil-Ladha**
Signed-off-by: Nikhil-Ladha <nikhilladha1999@gmail.com>

**Description**

This PR updates the careers page design.

**Note for reviewers**
Hope that it looks good 😁 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Review by Jashpatel1 — CHANGES_REQUESTED** [review]
Great change. Just a few things:
Lee's image of CCOSS looks stretched horizontally.
The transitions(shadows) come up a bit late I guess.
I didn't understand why the size of full-time opportunities was reduced horizontally if only 3 were to be fit in a single row.
Button Link in join the community section is incorrect.
Unused imports

**Comment by Nikhil-Ladha** [comment]
> The transitions(shadows) come up a bit late I guess.
It's merely 0.3s. 

> I didn't understand why the size of full-time opportunities was reduced horizontally if only 3 were to be fit in a single row.

The size is not reduced the card is made in that way, that so as to adapt to the size of content within it.
4 looks too much cozy, so kept them as 3 as well.

> Button Link in join the community section is incorrect.

Ahh, yes forgot to update that.

**Comment by Nikhil-Ladha** [comment]
The unused import for path, was there from before. I think few components still, pass it but it's commented in the actual code.

File: src/sections/Careers/index.js [hunk 1]
```diff
@@ -12,54 +13,60 @@ const CareersPage = () => {
...
+                        </p>
+                        <p>
+                            We believe that all contributors should expect and be part of a safe and
+                            friendly environment for constructive contribution. We can more
+                            effectively and successfully compare and challenge different ideas to find
+                            the best solutions for advancement, while building the size, diversity,
+                            and strength of our community.
+                        </p>
+                    </div>
+                    <div className="opensource-section-img">
```
Comment (Tanuj22) [inline src/sections/Careers/index.js:57]: Should we make a component out of this section image as the prop so we can use it in other places also?

**Review by Tanuj22 — COMMENTED** [review]
Rest LGTM!

File: src/sections/Careers/index.js (hunk 1 above)
Comment (Nikhil-Ladha) [reply src/sections/Careers/index.js:57]: I don't think that's needed. The usage of images varies a lot. And creating a component for it, won't be beneficial at all.

Comment (Tanuj22) [reply src/sections/Careers/index.js:57]: Yeah makes sense

**Review by Jashpatel1 — APPROVED** [review]
Looks good. @Tanuj22 should we proceed with merging or you fixing the failing tests?

**Comment by Nikhil-Ladha** [comment]
> Looks good. @Tanuj22 should we proceed with merging or you fixing the failing tests?

@Jashpatel1 fixed now.

**Pull request merged**